app/scripts/lib/app-start.js

```diff
@@ -84,15 +84,11 @@ define(function (require, exports, module) {
 
   Start.prototype = {
     startApp () {
-      // fetch both config and translations in parallel to speed up load.
-      return p.all([
-        this.initializeConfig(),
-        this.initializeL10n(),
-        this.initializeInterTabChannel()
-      ])
-      .then(this.testLocalStorage.bind(this))
-      .then(this.allResourcesReady.bind(this))
-      .fail(this.fatalError.bind(this));
+      return p()
+        .then(this.initializeDeps.bind(this))
+        .then(this.testLocalStorage.bind(this))
+        .then(this.allResourcesReady.bind(this))
+        .fail(this.fatalError.bind(this));
```

Comment:
And tangentially, this may have actually been working before, but I just tested and the STRs outlined in #2061 show a 500 page now.